assets/css/grammar.css

```diff
@@ -1,65 +1,40 @@
-/* no line numbering in syntax outlines */
```

Comment:
Deleted extraneous styles.  Moved heading and numbering styles into their own file.  Adding documentation comments.